scrapy/exporters.py

```diff
@@ -200,7 +200,7 @@ def _build_row(self, values):
             try:
                 yield to_native_str(s)
             except TypeError:
-                yield to_native_str(repr(s))
+                yield to_native_str(str(s))
```

Comment:
This was introduced in one of my latest commits to PY3 exporters, I think using `str` is indeed a better fallback behavior than `repr` for CSV.

This replicates better the behavior of csv.writer.writerow, which is what I originally intended:

```
>>> import csv, io, datetime
>>> b = io.BytesIO()
>>> w = csv.writer(b)
>>> w.writerow([datetime.datetime.now()])
28L
>>> b.getvalue()
'2016-01-27 16:01:04.673800\r\n'
```

Now that I think of it, I wonder what would happen if we just `yield s` on TypeError and leave off the `to_native_str`.